src/query/anonymization.c

```diff
@@ -574,6 +574,15 @@ typedef struct CollectMaterialContext
   char material[MAX_SEED_MATERIAL_SIZE];
 } CollectMaterialContext;
 
+static void normalize_function_name(char *func_name)
+{
+  /* TODO: add more cases. */
+  if (strcmp(func_name, "date_part") == 0)
+  {
+    strcpy(func_name, "extract");
```

Comment:
This makes me feel a bit queasy, as it works only because of the lengths of these strings, but I'm not sure if doing the dance with pfree/palloc/strcpy/whatnot to produce a proper new instance is worth it.

Reply:
Please add a comment that it is safe to do so because the length is shorter than the original one.